firebase-auth/src/androidMain/kotlin/dev/gitlive/firebase/auth/credentials.kt

```diff
@@ -104,6 +101,9 @@ actual class PhoneAuthProvider(val android: com.google.firebase.auth.PhoneAuthPr
                 }
             }
 
+            override fun onCodeAutoRetrievalTimeOut(verificationId: String) {
+            }
```

Comment:
we'll need to fail the CompletableDeferred here too otherwise it will hang?

Reply:
I do not think we should make a response.complete() in onCodeAutoRetrievalTimeOut(), because the response can be completed only once and we would not be able to make response.complete() after user input.
Maybe we could add a function inside PhoneVerificationProvider: onCodeAutoRetrievalTimeOut() ?

Reply:
you don't want to complete, you fail it with the appropriate exception